build_tools/cmake/iree_hal_cts_test_suite.cmake

```diff
@@ -87,6 +87,9 @@ function(iree_hal_cts_test_suite)
       "--iree-hal-target-backends=${_RULE_COMPILER_TARGET_BACKEND}"
       ${_RULE_COMPILER_FLAGS}
     )
+    if(_RULE_COMPILER_TARGET_BACKEND STREQUAL "llvm-cpu")
+      list(APPEND _TRANSLATE_FLAGS "--iree-llvmcpu-target-cpu=generic")
+    endif()
```

Comment:
Add to `COMPILER_FLAGS` at the call sites instead of adding a branch to the common function

* https://github.com/iree-org/iree/blob/main/runtime/src/iree/hal/drivers/local_sync/cts/CMakeLists.txt
* https://github.com/iree-org/iree/blob/main/runtime/src/iree/hal/drivers/local_task/cts/CMakeLists.txt

Reply:
done.